### DISCUSSION TITLE
AP name

### DISCUSSION THREAD

**Discussion by bmisra01** [joshschmelzle/lswifi#44] [General]
I assume each wlan vendor has custom IE for AP name. What vendors do you support and how to add a new one?

Thanks 
Bhupinder

**Comment by joshschmelzle** [comment]
Hi Bhupinder,

Yes, use lswifi with the `--ap-names` argument. I have implemented support for a few already-

- Aruba
- Cisco CCX1
- Ruckus
- Aerohive / Extreme
- WiNG / Extreme
- Juniper Mist

If you want a new one added, please send me a beacon with the ap name present.

**Comment by joshschmelzle** [comment]
Locking this conversation. Arista AP name parsing implemented in v0.1.35.
